### ISSUE TITLE
WCAG Compliance / Accessibility 

### ISSUE THREAD

**Issue by nathan6137** [istvan-ujjmeszaros/bootstrap-touchspin#88]
I am getting the following WCAG error when using the plugin

> This button element does not have a name available to an accessibility API. Valid names are: title attribute, element content.

Is there a way to pass a title for the up / down buttons?

I tried to use `buttondown_txt` property but it's ignored if you are using `verticalbuttons: true` ...
